packages/editor/src/components/visual-editor/index.js

```diff
@@ -272,7 +272,7 @@ function VisualEditor( {
 			'is-layout-flow': ! themeSupportsLayout,
 		},
 		themeSupportsLayout && postContentLayoutClasses,
-		align && `align${ align }`
+		align && ! [ 'wide', 'full' ].includes( align ) && `align${ align }`
```

Comment:
Any reason to keep left/right/center alignments?

Reply:
Mostly out of caution as post content blocks that use left/right alignment look quite different to a center aligned column and the reported issue was to do with wide/full alignments. Happy to remove this altogether if we don't want the left/right alignment.

I believe @jasmussen and @richtabor have more thoughts about the behaviour here, so I'll hold off on any updates until they get a chance to look at this.

From my perspective I think there are good arguments both ways between attempting to display some alignments in the abstracted (template preview off) mode, and skipping them entirely. I don't mind which way we go with it, so I'll leave this PR open for now until there's a chance for more feedback on the approach.

TL;DR — let me know if anyone would like to remove the left/right/center alignments here and I'll remove them 🙂

Reply:
I think it would make sense to leave left/right alignments because they don't cause problems 😅 and so we can have _some_ degree of WYSIWYG.

Reply:
Personally, I'm for consistency here. Left/right alignments for instance cause the whole thing to be "float" displayed, which IMO doesn't matter in this context at all and may actually create bugs. 

So instead of a half baked WYSIWYG, I think we should rather embrace that this is an abstracted view of the post editor and not meant to be full WYSIWYG in terms of layout.

Reply:
Oh hang on, actually I don't think there would be right/left align classes here; the Content block should only have wide/full enabled. 

When we set a constrained layout to justify left/right it doesn't use alignright or alignleft classes (or floats), it uses `margin-left: auto` or `margin-right: auto`. So it should be fine to remove the alignments here altogether! It won't make any difference to content justification.

Reply:
Sounds good! I'll remove the align classes here 👍

Reply:
Done in https://github.com/WordPress/gutenberg/pull/66352/commits/6d8f1cfba9d1eed0536a0a4b14d035e638051251

Reply:
thanks